etc/config.sample.toml

```diff
@@ -71,10 +71,14 @@
 
   # CacheMaxMemorySize is the maximum size a shard's cache can
   # reach before it starts rejecting writes.
+  # Valid size suffixes are k, m, or g (case insensitive).
+  # Values without a size suffix (e.g. 30k, 20m, 10g) are in bytes.
```

Comment:
"without a size suffix" - should it be 30, 20, 10 in that case? Or maybe just say that all values represent bytes.

To reduce any other confusion, it would help if this comment clarified k means 1000 and not 1024.

Reply:
I'm not sure that's the case. Since the sizes are byte shifts, then `1 << 10` is 1024. I'll see if I can clarify this comment though since what it's trying to say is that 30k, 20m, and 10g have size suffixes.

Reply:
You're right, it's 1024 and not 1000. Should still be clarified for users.

Reply:
I updated the configuration file to include a note that 1024 = 1k. That should be good enough for someone to extrapolate that these are 1024 and not 1000.